MediaGalleryUi/composer.json

```diff
@@ -9,6 +9,7 @@
         "magento/module-store": "*",
         "magento/module-media-gallery-ui-api": "*",
         "magento/module-media-gallery-api": "*",
+        "magento/module-media-gallery-synchronization-api": "*",
```

Comment:
I don't think MediaGalleryUi should depend on synchronization, please reverse the dependency